compiler/ilgen/OMRVirtualMachineOperandArray.hpp

```diff
@@ -73,6 +73,12 @@ class VirtualMachineOperandArray : public TR::VirtualMachineState
     * @param numOfElements the number of elements in the array
     */
    VirtualMachineOperandArray(TR::MethodBuilder *mb, int32_t numOfElements, TR::IlType *elementType, TR::VirtualMachineRegister *arrayBase);
+   /**
+    * @brief public constructor, must be instantiated inside a compilation because uses heap memory
+    * @param mb TR::MethodBuilder object of the method currently being compiled
+    * @param numOfElements the number of elements in the array

```

Comment:
Please add docs for the other parameters too: elementType, arrayBase, arrayBaseName